src/main/webapp/app/shared/markdown-editor/markdown-editor.component.ts

```diff
@@ -77,7 +77,7 @@ export class MarkdownEditorComponent implements AfterViewInit {
     @Input() markdown?: string;
     @Input() editorMode = EditorMode.NONE;
     @Input() showLineNumbers = false;
-    @Output() markdownChange = new EventEmitter<string>();
+    @Output() markdownChange = new EventEmitter<{}>();
```

Comment:
Are you sure you want to use an empty object type?

Reply:
I'm also not sure here, but as it only emits that something has changed, but not what I guess it should be fine.